rocAL/include/pipeline/tensor.h

```diff
@@ -267,6 +267,7 @@ class TensorInfo {
     bool is_metadata() const { return _is_metadata; }
     void set_roi_ptr(unsigned* roi_ptr) { _roi.reset_ptr(roi_ptr); }
     void copy_roi(void* roi_buffer) { _roi.copy(roi_buffer); }
+    std::shared_ptr<std::vector<float>> get_sample_rates() const { return _sample_rates; }  //!< The number of samples of audio carried per second
```

Comment:
please return as reference to avoid additional copy

Reply:
Here the vector is passed as shared pointer, hence it will not be copied, returning reference leads to errors as it is a const function